Eatery Blue/UI/Profile/ProfileLoginModelController.swift

```diff
@@ -105,32 +45,19 @@ class ProfileLoginModelController: ProfileLoginViewController {
     }
 
     private func attemptLogin() {
-        guard isLoginEnabled else {
-            return
-        }
-
         isLoggingIn = true
         view.endEditing(true)
-        updateErrorMessage(nil)
         updateLoginButtonFromState()
         AppDevAnalytics.shared.logFirebase(AccountLoginPayload())
 
         Task {
-            do {
-                let sessionId = try await Networking.default.sessionId.fetch(maxStaleness: .infinity)
+            if let sessionId = KeychainAccess().retrieveToken() {
                 delegate?.profileLoginModelController(self, didLogin: sessionId)
-
-            } catch GetAPIError.loginFailed {
-                updateErrorMessage("NetID and/or password incorrect, please try again")
-
```

Comment:
Not sure if these line breaks after the error function is convention

Reply:
wdym? i think i deleted them